src/main/java/net/snowflake/ingest/connection/ServiceResponseHandler.java

```diff
@@ -205,6 +256,20 @@ private static void handleExceptionalStatus(
               apiName,
               requestId == null ? "" : requestId.toString());
           throw new BackOffException();
+        case HttpStatus.SC_UNAUTHORIZED:
+          LOGGER.warn(
+              "401 Status hit from {}, requestId:{}",
+              apiName,
+              requestId == null ? "" : requestId.toString());
+          if (authRetries++ < Constants.MAX_TOKEN_REFRESH_FOR_UNAUTHORIZED_RETRY) {
+            requestBuilder.refreshToken();
+            requestBuilder.addToken(request);
+            LOGGER.info("Authorization fail, retries={}", authRetries);
```

Comment:
Lets keep this WARN, instead of info
`"Authorization failed, refreshing Token succeeded, retries={}"  `

Reply:
Fixed.